tests/pandas/test_grid_pandas.py

```diff
@@ -0,0 +1,1279 @@
+import numpy as np
+import pandas as pd
+import pytest
+import typeguard as tg
+
+from mesa_frames import GridPandas, ModelDF
+from tests.pandas.test_agentset_pandas import (
+    ExampleAgentSetPandas,
+    fix1_AgentSetPandas,
```

Comment:
You could add a noqa and the Ruff error code, as in https://github.com/projectmesa/mesa/blob/40dc2233d7652a3910e4fb06a0565e39e241845c/tests/test_agent.py#L105, to silence Ruff.

Reply:
I tried right now but it gives me F811 in every method Redefinition of unused `fix1_AgentSetPandas` from line 9.
I would leave this to avoid ignoring errors in every method